infrastructure_files/docker-compose.yml.tmpl

```diff
@@ -57,4 +57,4 @@ services:
 volumes:
   wiretrustee-mgmt:
```

Comment:
this should be updated to netbird-mgmt, but because docker doesn't provide an API to rename an existing volume, we need to update the template files to cover the case when there is an existing wiretrustee-mgmt directory and keep using it.

Reply:
done